package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "screener-runner",
-  "version": "0.11.9",
+  "version": "0.11.10",
```

Comment:
@eugenekatsov no need to update version - will get updated during publish process